eclass/ros-catkin.eclass

```diff
@@ -163,16 +163,19 @@ ros-catkin_src_prepare() {
 	append-cxxflags '-std=c++11'
 }
 
+ROSPREFIX="/opt/ros/${ROSDISTRO}"
```

Comment:
it works fine here

as for the problem with changing this like that, the one I mentioned is a quite major one